kubernetes/src/main/java/io/kubernetes/client/openapi/JSON.java

```diff
@@ -318,7 +320,7 @@ public void write(JsonWriter out, java.sql.Date date) throws IOException {
                         if (dateFormat != null) {
                             return new java.sql.Date(dateFormat.parse(date).getTime());
                         }
-                        return new java.sql.Date(ISO8601Utils.parse(date, new ParsePosition(0)).getTime());
```

Comment:
Why are we changing the date parser?

Reply:
it's a re-play of https://github.com/kubernetes-client/java/pull/366